src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceApiServiceSpec.scala

```diff
@@ -281,7 +281,7 @@ class WorkspaceApiServiceSpec extends BaseServiceSpec with WorkspaceApiService w
           //generally this is not how we want to treat the response
           //it should already be returned as JSON but for some strange reason it's being returned as text/plain
           //here we take the plain text and force it to be json so we can get the test to work
-          assert(WorkspaceCreate.isProtected(entity.asString.parseJson.convertTo[WorkspaceResponse].workspace))
+          assert(entity.asString.parseJson.convertTo[UIWorkspaceResponse].workspace.get.isProtected)
```

Comment:
This is testing GET, not post, so it needs to convert to UIWorkspaceResponse and not use WorkspaceCreate.